app/components/qualification_summary_component.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+class QualificationSummaryComponent < ViewComponent::Base
+  include ActiveModel::Model
+
+  attr_accessor :qualification
+
+  delegate :awarded_at, :details, :name, :type, to: :qualification
+
+  def rows
+    return itt_rows if name == "Initial teacher training (ITT)"
+
+    [
+      { key: { text: "Awarded" }, value: { text: awarded_at.to_fs(:long_uk) } },
+      {
+        key: {
+          text: "Certificate"
+        },
+        value: {
+          text:
+            link_to(
+              "Download #{type.to_s.upcase} certificate",
+              certificate_path(type),
+              class: "govuk-link"
+            )
+        }
+      }
+    ]
+  end
+
+  def title
+    name
+  end
+
+  def itt_rows
+    [
+      {
+        key: {
+          text: "Qualification"
+        },
+        value: {
+          text: details.qualification.name
+        }
+      },
+      { key: { text: "ITT provider" }, value: { text: details.provider.name } },
+      {
+        key: {
+          text: "Training type"
+        },
+        value: {
+          text: details.programmeType
```

Comment:
I think ideally the case of attributes/keys coming out of the API client would be consistent. Can we run a transform_keys on hashes prior to them being passed in as `details`?